resources/scss/componets/01_basics.scss

```diff
@@ -0,0 +1,131 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
```

Comment:
I think we don't need this cause may be bootstrap already have it. 
Please check and update based on that or let me know if I am wrong

Reply:
i remove this code now